routers/api/v1/convert/convert.go

```diff
@@ -45,6 +45,11 @@ func ToCommit(c *git.Commit) *api.PayloadCommit {
 		committerUsername = committer.Name
 	}
 	verif := models.ParseCommitWithSignature(c)
+	var signature, payload string
+	if verif.Verified {
```

Comment:
Should be  c.Signature != nil because Verified could be false because we didn't validated the sign but the commit is signed.